kuzzleio/kuzzle-admin-console
# [KZL-1439] Update user mapping

:warning: Depends on #636 

## What does this PR do ?
Migrates the Update User Mapping page to Bootstrap v4

### How should this be manually tested?
Add some properties to the user mapping and ensure they have been correctly saved.
